packages/commonwealth/client/scripts/views/sublayout_header.tsx

```diff
@@ -68,9 +72,8 @@ export const SublayoutHeader = ({
             iconName="dotsVertical"
             iconButtonTheme="black"
             onClick={() => {
-              app.sidebarToggled = false;
+              toggleSidebar(false);
               app.mobileMenu = app.mobileMenu ? null : 'MainMenu';
-              app.sidebarRedraw.emit('redraw');
```

Comment:
we can get rid of event emitter now because zustand is responsible to rerender component 💪

Reply:
This is perfect!